models/POP/readme.rst

```diff
@@ -175,11 +131,17 @@ It is run by the setup scripts.
 |                     | that you need to set in order to build and run cases. You |
 |                     | must read this file carefully and configure the variables |
 |                     | to match your needs. The assignment of the ``stagedir``   |
-|                     | variable in this script should match the assignment of    |
-|                     | the ``stagedir`` variable in                              |
-|                     | ``copy_POP_JRA_restarts.py``.                             |
+|                     | variable in this script should match the directory path   |
+|                     | where the restarts from the GDEX were downloaded/stored.  |
 +---------------------+-----------------------------------------------------------+
 
+One set of variables to make sure are set correctly in this script are related to
+the SourceMods, which  enable POP to recompute the barotropic velocity to prevent the
+barotropic solver from crashing. You can find and download the SourceMods available
+for POP-DART `here <https://www.image.ucar.edu/pub/DART/CESM/>`_. Ensure that the
+environment variable use_SourceMods is set to TRUE and the environment variable
+SourceModDir matches the directory path where the SourceMods were downloaded/stored.
+
```

Comment:
use_SourceMods is set in DART_params.csh but not used. The scripts just check for SourceMods in the users home directory, so it if you are adding docs about sourcemods in this pull request, probably best to match what the script does. 

https://github.com/NCAR/DART/blob/e2188646b97573f198c66f7ea63482ebc34afa11/models/POP/shell_scripts/cesm2_1/setup_CESM_hybrid_ensemble.csh#L350-L353

https://github.com/NCAR/DART/blob/e2188646b97573f198c66f7ea63482ebc34afa11/models/POP/shell_scripts/cesm2_1/setup_CESM_perfect_model.csh#L350-L353

This is my suggestion,  obviously Ben is a better reference for explaining what the source mods do. 

```suggestion
SourceMods are required to enable POP to recompute the barotropic velocity to prevent the
barotropic solver from crashing. You can find and download the SourceMods available
for POP-DART `here <https://www.image.ucar.edu/pub/DART/CESM/>`_. 

Put the SourceMods in your home directory:

~/${cesmtag}/SourceMods 

```